CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 - Major form fixes [#2000](https://github.com/opendatateam/udata/pull/2000)
 - Improved admin errors handling: visual feedback on all errors, `Sentry-ID` header if present, hide organization unauthorized actions [#2005](https://github.com/opendatateam/udata/pull/2005)
 - Expose and import licenses `alternate_urls` and `alternate_titles` fields [#2006](https://github.com/opendatateam/udata/pull/2006)
+- Switch from a "full facet clearing" to a "by term clear" approach in search facets [#2014](https://github.com/opendatateam/udata/pull/2014)
```

Comment:
choose between "clear" and "clearing" ;-)

Reply:
maybe "reset" 🤔

Reply:
:+1: for "reset" :+1: